process.c

```diff
@@ -8570,19 +8570,87 @@ proc_warmup(VALUE _)
 /*
  * Document-module: Process
  *
- *   The module contains several groups of functionality for handling OS processes:
- *
- *   * Low-level property introspection and management of the current process, like
- *     Process.argv0, Process.pid;
- *   * Low-level introspection of other processes, like Process.getpgid, Process.getpriority;
- *   * Management of the current process: Process.abort, Process.exit, Process.daemon, etc.
- *     (for convenience, most of those are also available as global functions
- *     and module functions of Kernel);
- *   * Creation and management of child processes: Process.fork, Process.spawn, and
- *     related methods;
- *   * Management of low-level system clock: Process.times and Process.clock_gettime,
- *     which could be important for proper benchmarking and other elapsed
- *     time measurement tasks.
+ * \Module \Process represents a process in the underlying operating system.
+ * Its methods support management of the current process and its child processes.
+ *
+ * == What's Here
+ *
+ * === Current-Process Getters
+ *
+ * - ::argv0: Returns the process name as a frozen string.
+ * - ::egid: Returns the effective group ID.
+ * - ::euid: Returns the effective user ID.
+ * - ::getpgrp: Return the process group ID.
+ * - ::getrlimit: Returns the resource limit.
+ * - ::gid: Returns the (real) group ID.
+ * - ::pid: Returns the process ID.
+ * - ::ppid: Returns the process ID of the parent process.
+ * - ::uid: Returns the (real) user ID.
+ *
+ * === Current-Process Setters
+ *
+ * - ::egid=: Sets the effective group ID.
+ * - ::euid=: Sets the effective user ID.
+ * - ::gid=: Sets the (real) group ID.
+ * - ::setproctitle: Sets the process title.
+ * - ::setpgrp: Sets the process group ID of the process to zero.
+ * - ::setrlimit: Sets a resource limit.
+ * - ::setsid: Establishes the process as a new session and process group leader,
+ *   with no controlling tty.
+ * - ::uid=: Sets the user ID.
+ *
+ * === Current-Process Execution
+ *
+ * - ::abort: Immediately terminates the process.
+ * - ::daemon: Detaches the process from its controlling terminal
+ *   and continues running it in the background as system daemon.
+ * - ::exec: Replaces the process by running a given external command.
+ * - ::exit: Initiates process termination by raising exception SystemExit
+ *   (which may be caught).
+ * - ::exit!: Immediately exits the process.
+ * - ::warmup: Notifies the Ruby virtual machine that the boot sequence
+ *   for the application is completed,
+ *   and that the VM may begin optimizing the application.
+ *
+ * === Child Processes
+ *
+ * - ::detach: Guards against a child process becoming a zombie.
+ * - ::fork: Creates a child process.
+ * - ::kill: Sends a given signal to processes.
```

Comment:
`kill` is not only for child processes.
But I can't say it should not be here since it is the typical usage...